handbook/engineering/distribution/tools/ghe_feeder.md

```diff
@@ -0,0 +1,149 @@
+# GHE feeder tool
+
+[GHE Feeder](https://github.com/sourcegraph/sourcegraph/tree/master/internal/cmd/ghe-feeder)
+
+Feeds repositories into a destination GHE instance. Input are files with owner/repo strings on each line and/or
+directories containing such files (see below for details about the expected input).
+
+The https://github.com/owner/repo repositories specified on the input lines are cloned and then pushed to a destination
+GHE instance. Every once in a while new orgs are created and the repos are added to those orgs.
+
+## Usage
+
+```shell script
+ghe-feeder -admin milton -token <xxxxxxx> -scratchDir clones -numWorkers 20 -limit 100000 -baseURL https://ghe.sgdev.org/api/v3 dir1 file2 file3 dir4
+```
+
+```
+ghe-feeder -help
```

Comment:
would this be a bit tedious to keep up to date? might be easier to just link to the `sourcegraph/sourcegraph` repo (https://github.com/sourcegraph/sourcegraph/tree/master/internal/cmd/ghe-feeder#usage) in this case (which is what I did for `resources-report`)

Reply:
good point

Reply:
i was planning to point the README to the handbook entry, but i can still point the usage the flags section in the code